libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/PluggableColumnBarCharts.tsx

```diff
@@ -67,6 +75,23 @@ export class PluggableColumnBarCharts extends PluggableBaseChart {
         );
     }
 
+    private addFiltersForColumnBar(source: IInsight, drillConfig: IImplicitDrillDown, event: IDrillEvent) {
+        const clicked = drillConfig.implicitDrillDown.from.drillFromAttribute.localIdentifier;
```

Comment:
I think I saw a function to retrieve this further on in the pr?

Reply:
Yes, it is one of the functions in ImplicitDrillDownHelper.ts .

Reply:
Fixed